app/models/converters/boolean_converter.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 class BooleanConverter < BaseConverter
-  TRUTHS = %w(true t yes ye y 1 on).freeze
-
   def self.convert(value)
     value = super(value)
-    value.blank? ? nil : TRUTHS.include?(value.to_s)
+
+    return nil if value.blank?
+    value.to_s.match(/\A(true|t|yes|ye|y|1|on)\z/i).present? ? true : false
```

Comment:
This is pretty awesome, but it might be something we would want to raise warnings around in the CSV message ultimately, maybe we can put a TODO comment or something in there. 

My ultimate goal would be to have the CSV that gets uploaded to be consistent. If there are this many freaking nuances to something characterized as true, the method whereby they are input is likely wrought with errors and the admins need to take a closer look at precisely what it is they are uploading.

Obviously this is just a nitpick for now, something to think about long-term.